src/containers/TriggerTemplate/triggertemplate.scss

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '~carbon-components/scss/globals/scss/vars';
+
+.bx--data-table-container {
```

Comment:
if all of the styles in this file are specific to TriggerTemplate can we nest them in a specific selector to ensure they don't affect other pages unintentionally?  Things like `.bx--data-table-container` and `.tableComponent` are used on multiple pages.